actionview/lib/action_view/digestor.rb

```diff
@@ -15,9 +15,10 @@ def self.before(target)
     class << self
       # Supported options:
       #
-      # * <tt>name</tt>   - Template name
-      # * <tt>finder</tt>  - An instance of <tt>ActionView::LookupContext</tt>
-      # * <tt>dependencies</tt>  - An array of dependent views
+      # * <tt>name</tt>         - Template name
+      # * <tt>format</tt>       - Template format
```

Comment:
This line is the only change.

I've just formatted the other lines to align the content. Let me know if I should revert that.

Reply:
@tenderlove ahh, shoot. I guess we did technically break public API here.

Reply:
I think we can make the format optional so it's backwards compatible

Reply:
@tenderlove Is there an easy default to set which I can do or is it a bigger task you'd like to take care of yourself?

Also, shouldn't this be merged in even before we set the default? Currently, as @kaspath said the API is broken but that does not mean we should not update the docs :)

Reply:
`nil` is probably fine as default? Can you open a PR setting it? I'll merge this one.

Reply:
Sure, will do.